atom/browser/browser_linux.cc

```diff
@@ -142,4 +142,39 @@ bool Browser::IsUnityRunning() {
   return unity::IsRunning();
 }
 
+void Browser::ShowAboutPanel() {
+  std::string app_name, version, copyright, icon_path, website;
+
+  GtkWidget* dialog = gtk_about_dialog_new();
```

Comment:
_(nit)_ We downcast the dialog widget a bunch of times here. Better to just do it once, e.g.

```cc
GtkDIalog* dialog = GTK_DIALOG(gtk_about_dialog_new());
```